string obj_class
float32 confidence
float32[] coordinates